src/cpu/meson.build

```diff
@@ -16,17 +16,17 @@ conf_data.set10('C_PER_PAGE_W_OR_X', per_page_w_or_x_pref.auto() or per_page_w_o
 core_selection = [
     # cpu_family selected_core        dynrec_define    target     unaligned  per-page
     #                                                             mem        W^X
-    [ 'x86_64',  ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86_64',  1,         1 ],
-    [ 'x86',     ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86',     1,         1 ],
-    [ 'x86_64',  ['dynrec'],          'C_DYNREC',      'X86_64',  1,         1 ],
-    [ 'x86',     ['dynrec'],          'C_DYNREC',      'X86',     1,         1 ],
-    [ 'aarch64', ['auto', 'dynrec'],  'C_DYNREC',      'ARMV8LE', 1,         1 ], # ARMv8+ (64-bit)
-    [ 'arm',     ['auto', 'dynrec'],  'C_DYNREC',      'ARMV7LE', 1,         1 ], # ARMv7+
-    [ 'armv6',   ['auto', 'dynrec'],  'C_DYNREC',      'ARMV4LE', 0,         0 ], # ARMv6 or older
-    [ 'ppc64',   ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors
-    [ 'ppc64le', ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors (little-endian)
-    [ 'ppc',     ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         0 ], # 32 bit PPC processors (big-endian)
-    [ 'mips',    ['auto', 'dynrec'],  'C_DYNREC',      'MIPSEL',  0,         0 ], # 32 bit MIPS processor
+    [ ['x86_64'],  ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86_64',  1,         1 ],
+    [ ['x86'],     ['auto', 'dyn-x86'], 'C_DYNAMIC_X86', 'X86',     1,         1 ],
+    [ ['x86_64'],  ['dynrec'],          'C_DYNREC',      'X86_64',  1,         1 ],
+    [ ['x86'],     ['dynrec'],          'C_DYNREC',      'X86',     1,         1 ],
+    [ ['aarch64'], ['auto', 'dynrec'],  'C_DYNREC',      'ARMV8LE', 1,         1 ], # ARMv8+ (64-bit)
+    [ ['arm'],     ['auto', 'dynrec'],  'C_DYNREC',      'ARMV7LE', 1,         1 ], # ARMv7+
+    [ ['armv6'],   ['auto', 'dynrec'],  'C_DYNREC',      'ARMV4LE', 0,         0 ], # ARMv6 or older
+    [ ['ppc64',   'powerpc64'],   ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors
+    [ ['ppc64le', 'powerpc64le'], ['auto', 'dynrec'],  'C_DYNREC',      'PPC64LE', 1,         1 ], # 64 bit PPC processors (little-endian)
+    [ ['ppc',     'powerpc'],     ['auto', 'dynrec'],  'C_DYNREC',      'POWERPC', 1,         0 ], # 32 bit PPC processors (big-endian)
```

Comment:
I'm puzzled by the 'ppc64' entry (not 'ppc64le', obviously). That's big-endian ppc64. I'm naturally big-endian ;) but if I read this right it's using the same JIT as little-endian and I'm pretty sure the JIT won't work running big as written (there's at least one endian issue I know of and have marked; there may be others). Perhaps safer to disable the JIT for that until it's been tested?

Reply:
I kind of want things to fail: if someone has a big endian PPC 64 (however rare that might be) then they will submit and issue  and from there, the community can rustle up experts who can fix up the code and run tests (like you ☺️ !) to get it working. 

(Otherwise these user might silently pass in the night, and be left with a poorly running normal core). 

Also, I don't think we have any CI or docker jobs running ppc64be, so no risk of ongoing failures.

Reply:
I re-aligned the table again.